### DISCUSSION TITLE
gmail smtp connection issue

### DISCUSSION THREAD

**Discussion by ryszard-suchocki** [postalserver/postal#3148] [Help with using Postal]
Dear all,
I am having trouble with Postal when sending emails to Google/Gmail. According to Postal, the connections end with the message: "No SMTP servers were available for gmail.com. Tried gmail-smtp-in.l.google.com, alt1.gmail-smtp-in.l.google.com, alt2.gmail-smtp-in.l.google.com, alt3.gmail-smtp-in.l.google.com, and alt4.gmail-smtp-in.l.google.com."
I have searched the reported bugs and disabled IPv6 in my LXC container. However, this did not resolve the issue. Communication with other SMTP servers works fine; only Gmail is problematic. I tried enabling extended logging, but the Worker logs show the same message: "(Net::OpenTimeout: Net::OpenTimeout)." However, this doesn't seem correct. I conducted a test using the "swaks" package (from the host/container where Docker is installed), and everything worked fine.
What can I do next to debug the issue?

**Comment by willpower232** [comment]
If you've removed all traces of IPv6 then it sounds like Google might be blocking connections from your server.

You could probably see if there were any `::` in `sudo netstat -lnp` to see if there is any trace of it left.

You can double check your general reputation with mxtoolbox.com but if they're blocking you specifically for their own reasons, there might not be anything you can do.

**Reply by ryszard-suchocki** [reply]
I checked two or three times. Sending mail from the host where Postal is installed is possible, and Gmail is not blocking the traffic. Is there any way to enable debugging for the Worker so I can see what it sends and receives from the server it connects to? I suspect the communication stops because of an unexpected response. My Postal installation is fresh—I installed it yesterday according to the documentation.

**Reply by willpower232** [reply]
Have you tried `postal logs smtp`? I'm not sure how much output there is off the top of my head.

There is probably an equivalent using `docker logs -f` but I couldn't tell you what the container name would be.
